docs/getting-started-guides/rkt/index.md

```diff
@@ -70,110 +65,147 @@ $ cat <<EOF >/etc/rkt/net.d/k8s_flannel_example.conf
+
+* Set the kubelet's `--rkt-stage1-image` flag, which tells the kubelet the stage1 image to use for every pod on the node. For example, `--rkt-stage1-image=coreos/rkt/stage1-coreos` selects the default systemd-nspawn stage1.
+* Set the annotation `rkt.alpha.kubernetes.io/stage1-name-override` to override the stage1 used to execute a given pod. This allows for mixing different container isolation mechanisms on the same cluster or on the same node. For example, the following (shortened) pod manifest will run its pod with the `fly stage1` to give the application -- the `kubelet` in this case -- access to the host's namespace:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kubelet
+  namespace: kube-system
+  labels:
+    k8s-app: kubelet
+  annotations:
+    rkt.alpha.kubernetes.io/stage1-name-override: coreos.com/rkt/stage1-fly
+spec:
+  containers:
+  - name: kubelet
+    image: quay.io/coreos/hyperkube:v1.3.0-beta.2_coreos.0
+    command:
+    - kubelet
+    - --api-servers=127.0.0.1:8080
+    - --config=/etc/kubernetes/manifests
+    - --allow-privileged
+    - --kubeconfig=/etc/kubernetes/kubeconfig
+    securityContext:
+      privileged: true
+[...]
+```
 
-After you created the cluster, you can start deploying apps to the cluster. For example here is how you can [deploy a simgle nginx app](/docs/user-guide/simple-nginx).
-More examples can be found in the [examples directory](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/).
+#### Notes on using different stage1 images
 
-### Known Issues and Differences
+Setting the stage1 annotation could potentially give the pod root privileges. Because of this, the `privileged` boolean in the pod's `securityContext` must be set to `true`.
 
-rkt and Docker have very different designs, as well as ACI and Docker image format.
-Users might experience some different experience when switching from one to the other.
-More information can be found [here](/docs/getting-started-guides/rkt/notes/).
+Use rkt's own [*contained network*](#use-rkts-contained-networking) with the LKVM stage1, because the CNI plugin driver does not yet fully support the hypervisor-based runtime.
 
-### Debugging
+### Known issues and differences between rkt and Docker
 
-Here are several tips in case you run into any issues.
+rkt and the default node container engine have very different designs, as do rkt's native ACI and the Docker container image format. Users may experience different behaviors when switching from one container engine to the other. More information can be found [in the Kubernetes rkt notes](/docs/getting-started-guides/rkt/notes/).
```

Comment:
should work in prod; if you checkout his branch and do `jekyll serve` you can browse it like that.